dm_cli/domain.py

```diff
@@ -6,13 +6,19 @@
 
 class Package:
     def __init__(
-        self, name: str, description: str = "", uid: UUID = None, is_root: bool = False
+        self,
+        name: str,
+        description: str = "",
+        uid: UUID = None,
+        is_root: bool = False,
+        meta: dict = None,
     ):
         self.name = name
         self.description = description
         self.uid = uid if uid else uuid4()
         self.is_root = is_root
         self.content: List[Union[Package, dict]] = []
+        self.meta: dict | None = meta if meta else {}
```

Comment:
```suggestion
        self.meta: Union[dict, None] = meta if meta else {}
```
in case we want compatibility with py3.8?

Reply:
fixed